features/step_definitions/general_steps.rb

```diff
@@ -18,7 +18,7 @@
 
 Given(/^I selected a project area "([^"]*)"$/) do |area_source|
   @app.project_area_selection_page.submit(
-    areasource: area_source
+    area: area_source
```

Comment:
Not sure about this change - why change the name of the key?

Reply:
Fixed in next push